lib/bindConnectors/bindMessage.js

```diff
@@ -76,11 +156,45 @@ module.exports = function (message, threadneedle, connectorConfig) {
 
 				threadneedle[methodName](body)
 
-				.done(resolve, reject);
+				.done(function (data) {
+					requestEnd(startTime, { connectorName, operationName: methodName, connectorVersion: connectorConfig.version, status: 'success' }, datadogInstance);
+					resolve(data);
+				}, function (data) {
+					if (process.env.ENABLE_LOGGING === 'true' && logger) {
```

Comment:
It would be better making this lib specific, i.e. `FALAFEL_ENABLE_LOGGING` or `FALAFEL_ENABLE_LOGS`

We similarly have it in threadneelde https://github.com/trayio/threadneedle/blob/master/lib/logger.js#L21